Dataset/examples/0.3-DRAFT_examples/DisProt_jsonld.json

```diff
@@ -1 +1 @@
-../../../DataCatalog/examples/0.3/DisProt_jsonld.json
\ No newline at end of file
+../../../DataCatalog/examples/0.3-DRAFT/DisProt_jsonld.json
```

Comment:
I suggest using a real link here rather than a json containing a link. A real link has the advantage that is clickable. What about creating a markdown page there, e.g., "Additional examples.md" and then a line such
```
DisProt DataCatalog example also includes a [Dataset set example](../../../DataCatalog/examples/0.3/DisProt_jsonld.json)
```

Reply:
It is a symlink and it should work with GitHub. At least it worked until it broke when the directories with examples were renamed. What you see here is a "content" of a symlink.

Reply:
Ok, I will accept then